apps/directory/src/stores/settingsStore.js

```diff
@@ -39,13 +36,26 @@ export const useSettingsStore = defineStore("settingsStore", () => {
     landingpage: initialLandingpage,
     pageSize: 12,
     i18n,
-    banner: undefined, //"<div style='background-color: #72f6b2; padding: 10px; text-align: center;'>I am in a banner</div>"
+    banner: ``,
+    footer: ``,
```

Comment:
I suppose a good default would help?

Reply:
I asked Dieuwke what would be prefered here.